hack/manifests/pod.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: container
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      app: container
+  template:
+    metadata:
+      labels:
+        app: container
+    spec:
+      containers:
+        - name: ubuntu
+          image: mcr.microsoft.com/mirror/docker/library/ubuntu:22.04
+          imagePullPolicy: Always
+          command: ["/bin/sleep", "3650d"]

```

Comment:
if you just want a no-op pod, use `mcr.microsoft.com/oss/kubernetes/pause:3.6`. it will pull faster, doesn't need a sleep command, and only does that one thing

Reply:
Yeah just wanted to have no-op. Thanks for the image.